scripts/README.md

```diff
@@ -0,0 +1,21 @@
+# Dev Scripts
+
+These are a collection of scripts that should be helpful for those developing
+on dendrite.
+
+- `./scripts/find-lint.sh` runs the linters against dendrite,
```

Comment:
this looks like a list that is going to get out of date in about 30 seconds.

I'd suggest sticking these descriptions in the comments at the top of each script. And possibly duplicating the env var descriptions to each one?